app/views/users/index.html.erb

```diff
@@ -1 +1,16 @@
-<h1>Here is a list of all users</h1>
\ No newline at end of file
+<div class="user-page">
+  <% @users.each do |user| %>
+    <div class="user-cont">
+      <div>
+        <img src="https://user-images.githubusercontent.com/37116322/222183427-3450696b-429a-4e9a-9e19-48bbe4b48edf.png" alt="profile">
+      </div>
+      <div class="user-info">
+        <h2><%= user.name %></h2>
+        <div>
+          <p>Number of posts: <%= user.posts_counter %></p>
+          <%= link_to 'View', "/users/#{user.id}" %>
+        </div>
+      </div>
+    </div>
+  <% end %>
```

Comment:
## [OPTIONAL]

- [ ] Nice work displaying users' information here but for best practice, I would suggest you the option below  instead of using `@users.each`
```html.erb
    # app/views/users/_user.html.erb

     #   HTML Code Here displaying
     #   user.photo, user.name and user.posts_counter

    # app/views/users/index.html.erb

    <%= render @users %>
```